site/src/Librecores/ProjectRepoBundle/Resources/config/services.yml

```diff
@@ -1,89 +1,94 @@
+# Learn more about services, parameters and containers at
+# http://symfony.com/doc/current/book/service_container.html
+imports:
+    - { resource: legacy_aliases.yml }
+
+
 services:
     _defaults:
         autowire: true
         autoconfigure: true
+        public: false
+
+    # makes classes in src/Librecores/ProjectRepoBundle available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    Librecores\ProjectRepoBundle\:
+        resource: '../../../ProjectRepoBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../../ProjectRepoBundle/{Entity,Repository,DataFixtures}'
+        # The class in the file Librecores\ProjectRepoBundle\DataFixtures\ORM\LoadDevTestData is named
+        # "LoadProjectData" and not as expected "LoadDevTestData", therefore it cannot be created a service for this
+        # class and is excluded above.
```

Comment:
good catch! Please rename the class in LoadDevTestData.php to LoadDevTestData to make the autoloader happy.